tests/test_stack.py

```diff
@@ -267,6 +273,139 @@ def test_update_sends_correct_request_no_notification(
         )
         mock_wait_for_completion.assert_called_once_with()
 
+    @pytest.mark.parametrize("function",  ["create", "update"])
+    @patch("sceptre.stack.Stack.launch_using_change_set")
+    def test_functions_calls_launch_using_change_set_when_required(
+            self, mock_launch_using_change_set, function
+    ):
+        self.stack._template = Mock(spec=Template)
+        self.stack._template.requires_change_set = True
+
+        self.stack.__getattribute__(function).__call__()
+
+        mock_launch_using_change_set.assert_called_once_with()
+
+    @patch("sceptre.stack.Stack.get_status")
+    @patch("sceptre.stack.Stack.launch_using_change_set")
+    def test_update_does_not_call_launch_using_change_set_when_non_existant(
```

Comment:
Nit: `existent`

Reply:
Not sure on the difference other tests also use existant too, but will update